packages/actor-init-query/test/HttpServiceSparqlEndpoint-test.ts

```diff
@@ -48,6 +49,16 @@ const argsDefault = {
 };
 
 describe('HttpServiceSparqlEndpoint', () => {
+  let originalCluster: typeof cluster;
```

Comment:
Is this a fix for #1106? If so, could we move this to a separate PR?

Reply:
Yep, sorry! Was experimenting with that last night since I needed a test execution environment.

Reply:
Will push reversion once the fix (https://github.com/comunica/comunica/pull/1109) is merged.
Or would you recommend to push it immediately?

Reply:
It's ok, we can wait :)